pkg/util/ec2/ec2.go

```diff
@@ -111,6 +111,19 @@ func IsRunningOn(ctx context.Context) bool {
 	return false
 }
 
+// GetHostAliases returns the host aliases from GCE
```

Comment:
```suggestion
// GetHostAliases returns the host aliases from AWS
```

Reply:
or `EC2`?

Reply:
🤔  Probably `EC2` but it sounds weird saying that. Maybe it should say `AWS IMDS` as we are pulling the data from [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html).